backend/server.py

```
@@ -539,22 +561,35 @@ def get_transcript():
-
-    latest_subtitle = max(subtitle_files, key=os.path.getctime)
-    transcript_text = clean_transcript(latest_subtitle)
+    transcript_text = None
+    latest_subtitle = None
 
-    # Optional: Clean up the file after reading
-    os.remove(latest_subtitle)
-
-    return jsonify({"transcript": transcript_text})
+    try:
+        subprocess.run(["yt-dlp", "--write-auto-sub", "--sub-lang", "en", "--skip-download",
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)